### PR TITLE
v0.0.2

### PR DESCRIPTION
**Pull request by i-like-robots**
Essentially this is a re-write so here are some screenshots and squashed into a single commit:

Dashboard:

screen shot 2017-01-17 at 12 13 40

Searching for a video:

screen shot 2017-01-17 at 12 13 47

Video edit screen with tags added:

screen shot 2017-01-17 at 12 14 12

- Adds an `/api` endpoint for Mongo CRUD operations
- React-tastic SPA - the contents of the Redux store can be synced with the API above
- Dashboard view displaying recently edited and latest videos
- Video view showing the video and with save/publish/delete controls

### CODE REVIEW

File: client/components/Video/VideoPublish.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		fetch(`/admin/${this.props.id}`, {
+			method: 'PUT',
+			credentials: 'same-origin'
+		})
+			.then(() => {
+				window.alert('Video published');
+			})
+			.catch((err) => {
+				window.alert('Sorry, an error occured');
+				window.console.error(err.toString());
```
Comment (commuterjoy) [inline client/components/Video/VideoPublish.js]: i guess stick on a TODO list raven/sentry

File: server/controllers/api/video.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+		})
+		.catch((err) => {
+			logger.error(err);
+			next(err);
+		});
+});
+
+router.delete('/:brightcoveId', (req, res, next) => {
+	const query = Video
+		.remove({ brightcoveId: req.params.brightcoveId })
```
Comment (commuterjoy) [inline server/controllers/api/video.js]: probably best practice for `remove` operations to really be `updates` with a `deleted = true` flag.

**Comment by commuterjoy** [comment]
👍 lovely stuff

File: server/controllers/api/video.js (hunk 2 above)
Comment (i-like-robots) [inline server/controllers/api/video.js]: 👍

**Pull request merged**